portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase

```diff
@@ -889,4 +889,144 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-128810. The content page must shows only the Web Content with tag when choose the Segment."
+	@priority = "4"
+	test ViewWebContentTitleOnPageAfterChooseTheSegment {
+		property portal.acceptance = "true";
+
+		task ("Add Segment via JSON") {
+			JSONSegmentsentry.addSegment(
+				fieldName = "First Name",
+				groupName = "Test Site Name",
+				operator = "equals",
+				segmentName = "Segment Name",
+				text = "user");
+		}
+
+		task ("Add a Web Content via JSON") {
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title 1");
+		}
+
+		task ("Add a Web Conten with tag via JSON") {
+			JSONWebcontent.addWebContent(
+				assetTagNames = "tag name",
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title 2");
+		}
+
+		task ("Add a dynamic Collections and select Web Content Article as Asset Entry Type") {
+			JSONAssetlist.addDynamicAssetListEntryWithItemType(
+				groupName = "Test Site Name",
+				itemSubtype = "Basic Web Content",
+				itemType = "Web Content Article",
+				title = "Dynamic Collection");
+		}
+
+		task ("Set up the Personalized Variation") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection");
+
+			AssetListsAdmin.addPersonalizedVariation(
+				itemSubtype = "Basic Web Content",
+				itemType = "Web Content Article",
+				segmentName = "Segment Name");
+
+			AssetListsAdmin.addTagsFilter(tagNameList = "tag name");
+		}
+
+		task ("View all web content shown in collection items") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.viewContent(
+				assetListTitle = "Dynamic Collection",
+				assetTitle = "Web Content Title 1,Web Content Title 2",
+				assetType = "Web Content Article");
+		}
+
+		task ("View only the second web content shown in collection items") {
```

Comment:
View only the web content with tag shown in collection items when select New Segment personalized variation